tests/integration/DatabaseUpdateTest.php

```diff
@@ -107,18 +107,18 @@ public function testDataIsStillThereAfterColumnUpdate() {
 
 	public function testDataIsSetToDefaultAfterColumnUpdate() {
 		$schemaTo = $this->schemaManager->createSchema();
-		$schemaTo->getTable( 'action_log' )->dropColumn( 'al_timestamp' );
+		$schemaTo->getTable( 'backend_impressions' )->dropColumn( 'datetime' );
 		$this->updateDatabaseBySchema( $schemaTo );
 
-		$this->factory->getConnection()->insert( 'action_log', [ 'al_username' => 'test' ] );
+		$this->factory->getConnection()->insert( 'backend_impressions', [ 'banner_id' => 'banner1' ] );
 
 		$this->factory->newUpdater()->update();
 
-		$entity = $this->entityManager->getRepository( 'WMDE\Fundraising\Entities\ActionLog' )->findOneBy( [ ] );
+		$entity = $this->entityManager->getRepository( 'WMDE\Fundraising\Entities\BackendImpressions' )->findOneBy( [ ] );
```

Comment:
I do not think that is much of an improvement given the test is fully isolated. When reading the test, for me the only difference would be that I'd also have to verify this additional code is correct. And one can argue that adding the condition binds to more code that we do not actually care about testing here, though that is a rather tenuous difference here.